core/src/main/java/com/eternalcode/core/util/EntityUtil.java

```diff
@@ -0,0 +1,34 @@
+package com.eternalcode.core.util;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Mob;
+
+public final class EntityUtil {
+
+    public static boolean is(Entity entity, Class<? extends Entity> entityClass) {
+        return is(entity.getClass(), entityClass);
+    }
+
+    public static boolean is(Class<?> clazz, Class<? extends Entity> entityClass) {
+        return entityClass != null && clazz != null && entityClass.isAssignableFrom(clazz);
+    }
+
+    public static boolean isMob(Entity entity) {
+        return isMob(entity.getClass());
+    }
+
+    public static boolean isMob(EntityType entityType) {
+        Class<? extends Entity> entityClass = entityType.getEntityClass();
+        return isMob(entityClass);
+    }
+
+    public static boolean isMob(Class<? extends Entity> entityClass) {
+        return is(entityClass, Mob.class);
+    }
+
+    private EntityUtil() {
```

Comment:
code style: konstruktory dajemy na górę

Reply:
Zauważyłem, że tutaj jest na dole to też dałem

https://github.com/EternalCodeTeam/EternalCore/blob/88f5fa7fbc4bda88aa3c5e94422ca9aa1746f461/core/src/main/java/com/eternalcode/core/util/ItemUtil.java